<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadialGaugeSettingWidget</class>
 <widget class="QWidget" name="RadialGaugeSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="previewGroupBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="RadialGauge" name="preview" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="scaleTab">
      <attribute name="title">
       <string>Scale</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="directionLabel">
         <property name="text">
          <string>Direction:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="direction">
         <property name="maximumSize">
          <size>
           <width>165</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Clockwise</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Counter Clockwise</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="rangeGroupBox">
         <property name="title">
          <string>Range</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="rangeMinLabel">
            <property name="text">
             <string>Minimum:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="rangeMin">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="rangeMaxLabel">
            <property name="text">
             <string>Maximum:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="rangeMax">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="scaleArcGroupBox">
         <property name="title">
          <string>Scale Arc</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QLabel" name="startAngleLabel">
            <property name="text">
             <string>Start:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="startAngle">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>359.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="sweepAngleLabel">
            <property name="text">
             <string>Sweep:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QDoubleSpinBox" name="sweepAngle">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>359.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ticksTab">
      <attribute name="title">
       <string>Ticks</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="tickCountGroupBox">
         <property name="title">
          <string>Max. Tick Count</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="majorLabel">
            <property name="text">
             <string>Major:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="majorTick">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="minorLabel">
            <property name="text">
             <string>Minor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="minorTick">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="labelGroupBox">
         <property name="title">
          <string>Labels</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QFontComboBox" name="font">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="fontLabel">
            <property name="text">
             <string>Font:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="fontSizeLabel">
            <property name="text">
             <string>Font Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="fontSize">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="needleTab">
      <attribute name="title">
       <string>Needle</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="valueLabel">
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="value">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>-999999999</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="needleTypeLabel">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="needleType">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Simple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magnet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wind Arrow</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="needleStyleLabel">
         <property name="text">
          <string>Style:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="needleStyle">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Style 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Style 2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="needleColor1Label">
         <property name="text">
          <string>Color1:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ColorButton" name="needleColor1">
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="needleColor2Label">
         <property name="text">
          <string>Color2:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="ColorButton" name="needleColor2">
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textColorLabel">
         <property name="text">
          <string>Text:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ColorButton" name="textColor">
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="backgroundColorLabel">
         <property name="text">
          <string>Background:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ColorButton" name="backgroundColor">
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>widget/colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>RadialGauge</class>
   <extends>QWidget</extends>
   <header>gauge/radialgauge.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
